ltr_tgs.txt

```diff
@@ -48,44 +40,38 @@
 |- align= "center
 |0-17-103||Waltham||Armstrong Ambulance Chair Car||2022.10/.05 OceanNora
 |- align = "center"
-|1-01-102||Hudson||To be Identified (Plowing Nashua NH)||2022.01.07 Dave2582
+|0-01-102||Hudson||To be Identified (Plowing Nashua NH)||2022.01.07 Dave2582
 |- align="center"
 |1-10-103||Boston||To be Identified - Transport/Delivery Service in/around Waltham||2020.12.28 ecps92
 |- align="center"
-|1-12-104||||Armstrong ALS backup channel||2017.05.29 N1ZYY
+|1-12-104||Boston||Armstrong ALS backup channel||2017.05.29 N1ZYY
 |- align="center"
-|1-12-105||||Armstrong BLS backup channel||2017.05.29 N1ZYY
+|1-12-105||Boston||Armstrong BLS backup channel||2017.05.29 N1ZYY
 |- align="center"
-|1-12-106||||Armstrong BLS/May also include Chair Cars||2017.11.03 ecps92
+|1-12-106||Boston||Armstrong BLS/May also include Chair Cars||2017.11.03 ecps92
 |- align="center"
-|1-01-107||Hudson||To be Identified (Plowing Nashua NH)||2018.01.10 N1ZYY
-|- align="center"
-|1-xx-127||||To be identified (Taxi or TowCo)||2017.05.29 N1ZYY
+|0-01-107||Hudson||To be Identified (Plowing Nashua NH)||2018.01.10 N1ZYY
 |- align="center"
 |0-01-132||Waltham||To be Identified (Plowing/Construction)||2022.11.22 N1ZYY
 |- align="center"
-|1-12-140||Waltham||To be Identified (Poss Taxi) Voice Inversion||2017.09.04 ecps92
-|- align="center"
-|1-12-141||Waltham||To be Identified (Poss Taxi) Voice Inversion||2017.09.04 ecps92
-|- align="center"
 |0-01-142||Waltham||Unknown Construction company||2020.01.31 ecps92
 |- align="center"
 |0-01-151||Hudson||PrideStar EMS ||2021.09.15 MASSBUSMARK
 |- align="center"
 |0-01-152||Hudson||PrideStar EMS - Chair Car Ops||2021.09.15 MASSBUSMARK
 |- align="center"
-|1-01-153||Hudson||To be Identified (Plowing)||2018.01.10 N1ZYY
+|0-01-153||Hudson||To be Identified (Plowing)||2023.01.06 N1ZYY
 |- align="center"
 |0-01-155||Waltham||Armstrong Ambulance ALS (911)||2021.01.28 MASSBUSMARK
 |- align="center"
 |0-01-156||Waltham||Armstrong Ambulance BLS/Transfer||2021.01.28 MASSBUSMARK
 |- align="center"
 |0-01-157||Waltham||Armstrong Ambulance Chair Car||2022.02.16 ecps92
 |- align="center"
-|x-01-158||Waltham||Unknown - Snow Plows (Boston and Suburbs)||2018.01.12 N1ZYY
+|0-01-158||Waltham||Unknown - Snow Plows (Boston and Suburbs)||2018.01.12 N1ZYY
 |- align="center"
-|x-09-159||Waltham||Unknown Chair Car company||2013.09.25 N1BHH
+|0-09-159||Waltham||Unknown Chair Car company||2013.09.25 N1BHH
 |- align="center"
-|x-01-170||Waltham||Unknown - Transportation Company||2017.06.29 ecps92
+|0-01-170||Waltham||Unknown - Transportation Company||2017.06.29 ecps92
 |- align="center"
-|x-xx-253||All Sites||Note: Several different TGs used for CW ID||2017.11.03 GaryS
+|x-xx-253||All Sites||Talkgroups used for CW ID||2017.11.03 GaryS
```

Comment:
This was possibly an unnecessary change. The point I was trying to make is that this is standard LTR behavior; only 001-250 are valid for "normal" talkgroups, and LTR controllers will typically key the repeater on their home channel and show 253 for the talkgroup when IDing.